scss/pixel-arts/coin.scss

```diff
@@ -0,0 +1,41 @@
+.nes-coin {
+  $coin-colors: (#060606,#ffd700, #daa520);
+  // prettier-ignore
+  $coin: (
+    (0,0,0,0,0,1,1,1,1,0,0,0,0,0),
+    (0,0,0,1,1,2,2,2,2,1,1,0,0,0),
+    (0,0,1,2,2,1,1,1,1,2,2,1,0,0),
+    (0,1,2,1,1,3,3,3,3,1,2,1,1,0),
+    (0,1,2,1,3,3,3,3,3,3,1,2,1,0),
+    (1,2,1,3,3,3,3,3,3,3,3,1,2,1),
+    (1,2,1,3,3,3,3,3,3,3,3,1,2,1),
+    (1,2,1,3,3,3,3,3,3,3,3,1,2,1),
+    (1,2,1,3,3,3,3,3,3,3,3,1,2,1),
+    (0,1,2,1,3,3,3,3,3,3,1,2,1,0),
+    (0,1,2,1,1,3,3,3,3,1,2,2,1,0),
+    (0,0,1,2,2,1,1,1,1,2,2,1,0,0),
+    (0,0,0,1,1,2,2,2,2,1,1,0,0,0),
+    (0,0,0,0,0,1,1,1,1,0,0,0,0,0),
+  );
+  $size: 6px;
+
+  position: relative;
+  display: inline-block;
+  width: $size * length(nth($coin, 1));
+  height: $size * length($coin);
+
+  &::before {
+    position: absolute;
+    top: -$size;
+    left: -$size;
+    content: "";
+    background: transparent;
+
+    @include pixelize($size, $coin, $coin-colors);
+  }
+
+  &.earned {
+    animation: rotateIn 1.4s ease-out forwards,
+    fadeOut 2s ease-out forwards;
+  }
```

Comment:
NES.css's icons does not provide animation.
so could you define it in your project? 🙏 

Because allowing one animation will increase endlessly. and the animation required by the user  is different.

Reply:
My thinking was that this animation could come with the library since it's more about the liveliness of the element and that sweet retro gaming feeling.

Although truth being said, there was not a lot of 3-D animations back in the day :P

I see your point, I kinda expected it so I made it on a separate commit :clipboard:

Reply:
_resolving as it is done_